### ISSUE TITLE
Arch Linux Support

### ISSUE THREAD

**Issue by btd1337** [sgpthomas/hourglass#51]
Arch Linux Support

I will support your application in Arch Linux. Please, consider to use 2 branchs: master (stable code), develop (development code).

Please add a section containing the following instruction:

## Arch Linux
Arch Linux users can find `Hourglass` under the name [hourglass-git](https://aur.archlinux.org/packages/hourglass-git/) in the **AUR**:

`$ aurman -S hourglass-git`

**Comment by nucleusfox** [comment]
Hi, I've just installed it on Arch (via yaourt) and Timer seems to be a complete crazy when select hours. It rings immediately if I set 1-3 hours, for example 01:00:00, as if it was 0 seconds. And it shows negative time when set 4 hours: 04:00:00 -> shows -2:02:04: and counts from there, so if I wait for enough time I guess it will count to 0 and rings after 2h 2min 4sec, not 4h. I suspect it's not a feature by design. Alarm works fine though.
